### DISCUSSION TITLE
Does the Vercel platform block Google Adsense clicks for hobby accounts?

### DISCUSSION THREAD

**Discussion by hissetahminleri** [vercel/community#5126] [Help] [answered]
### Summary

I created a new site using next js for the first time. I know that Google Adsense will not be used for fair purposes. But I used Google Adsense component to try it.
But it has not been working as I wanted for 13 days.
According to Google Adsense data, ad impressions were 60,081 but the number of clicks was 0. But Google Adsense does not count ad clicks. According to Google Analytics data, ad impressions were 57660. The number of people clicking on ads is 312, but the number of ads clicked is 11. Normally, the number of people clicking on an ad cannot be less than the ad clicked.

### Steps to Reproduce

```
// components/AdComponent.js
"use client";
import React, { useEffect } from "react";

const Adsense = () => {
  useEffect(() => {
    if (window) {
      const script = document.createElement("script");
      script.src =
        "https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-xxx";
      script.async = true;
      script.crossOrigin = "anonymous";
      document.body.appendChild(script);

      script.onload = () => {
        if (window.adsbygoogle && process.env.NODE_ENV !== "development") {
          window.adsbygoogle.push({});
        }
      };
    }
  }, []);

  return (
    <ins
      className="adsbygoogle"
      style={{ display: "block", width: "100%", height: "auto" }}
      data-ad-client="ca-pub-xxx"
      data-ad-slot="xxx"
      data-ad-format="auto"
      data-full-width-responsive="true"
    ></ins>
  );
};

export default Adsense;
```
![Adsense data](https://github.com/vercel/community/assets/153314597/5edb14f7-fdb6-4a40-84c2-69750ebc6d1c)
![Analytics click data](https://github.com/vercel/community/assets/153314597/67c8c12d-ef76-4bfd-8320-ee9d63de4d20)

**Comment by riyazweb** [comment]
yeah may be i too got the same problem

**Comment by amyegan** [comment, marked as answer]
Commercial use is prohibited on the Hobby plan so advertisements cannot be used on these deployments: https://vercel.com/docs/limits/fair-use-guidelines#commercial-usage

Additionally, if the site uses a `*.vercel.app` domain then you may not be able to verify the domain. For this to work, you should upgrade to a [Pro plan](https://vercel.com/docs/accounts/plans/pro) and [add your own custom domain](https://vercel.com/docs/projects/domains/add-a-domain).
